keras_nlp/samplers/sampler.py

```diff
@@ -110,11 +110,13 @@ def body(prompt, state, index):
             # Compute the softmax distribution for the next token.
             logits, state = next(prompt, state, index)
             probabilities = keras.activations.softmax(logits)
-
             # Compute the next token.
             next_token = self.get_next_token(probabilities)
             # Don't overwrite anywhere mask is True.
             next_token = tf.cast(next_token, prompt.dtype)
+            # Ensure shape is `[None]`, otherwise it causes issues after
+            # converting to TFLite.
+            next_token = tf.ensure_shape(next_token, [None])
```

Comment:
Do you have an example of where this is causing a failure? Very curious why this is needed.

Is there someway we can test the failure without needing to depend on TFLite you think?